pkg/logs/decoder/line_handler.go

```diff
@@ -57,7 +57,7 @@ func (h *SingleLineHandler) run() {
 	for line := range h.lineChan {
 		h.process(line)
 	}
-	h.outputChan <- newStopOutput()
+	close(h.outputChan)
```

Comment:
go noob question: when we close a channel, it first delivers all its content before actually closing, right?

Reply:
this is correct